WorkingInR/2_First_Steps_in_R.Rmd

```diff
@@ -337,24 +363,28 @@ df.tibble <- as_tibble(df)
 df.tibble 
 ```
 
-Here is a tibble called `penguins` which is part of the package `palmerpenguins`. You will certainly meet this tibble again and again on this website as it provides a convenient set of variables and observations well-suited for illustrating many purposes.
+Here is a tibble called `penguins` which is part of the package `palmerpenguins`. 
+You will certainly meet this tibble again and again on this website as it provides a convenient set of variables and observations well-suited for illustrating many purposes.
 
 ```{r penguins-tibble, echo=TRUE}
 penguins
 ```
 
 ### Lists
 
-A list is an object that contains values of **various** data types. The object `my.list` described below is a list that contains 6 data elements: some are logical, others are numerical or character. Note that a list is created using the function `list()` that concatenates the elements while preserving the data type.
+A list is an object that contains values of **various** data types. 
+The object `my.list` described below is a list that contains 6 data elements: some are logical, others are numerical or character.
+Note that a list is created using the function `list()` that concatenates the elements while preserving the data type.
 
-Read more about lists [here](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/list)
+Read more about lists [here](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/list).
 
 ```{r list, echo=TRUE}
 my.list <- list("one", 2, "three", TRUE, FALSE, -42)
 my.list 
 ```
 
```

Comment:
For the list chapter, you can again merge the creation of the list and directly give each list element a name. And not do it first without and then with names. Good practise and saves some space.

You could also give "meaningful" names.
c could be temperature
f could be matrix
not so sure about the others...